.buildkite/longtests.pipeline.yml

```diff
@@ -1,6 +1,6 @@
 # Copied from pipeline.yml.
```

Comment:
Hmm... could you file a new issue to stop copying these parts of YAML files but rather break them out to a separate YAML file and combine them when uploading the pipelines via `cat $pipeline-common $pipeline | buildkite-agent pipeline upload`?

Reply:
#2972

Reply:
Thanks!